SampleGetApi/ReportsHelper/ReportUtils.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Xml.Linq;
+using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+
+namespace SampleGetApi.ReportsHelper;
+
+	public class ReportUtils
+	{
+    public static ExtentReports? ExtentReport;
+
+    public static void Initalize()
+		{
+        var reportFolder = CreateReportFolder();
+
+        if (reportFolder != null || reportFolder != "")

```

Comment:
String.IsNull OrEmtpy

Reply:
Will change the condition it has to be Empty.